pkg/container-collection/container-collection.go

```diff
@@ -198,6 +198,33 @@ func (cc *ContainerCollection) LookupPIDByPod(namespace, pod string) map[string]
 	return ret
 }
 
+// LookupOwnerReferenceByMntns returns the owner reference of the container
+// identified by the mount namespace, or zero if not found
+func (cc *ContainerCollection) LookupOwnerReferenceByMntns(mntns uint64) map[string]string {
```

Comment:
What about returning `c.OwnerReference` directly and avoiding the conversion to map?

Reply:
Yeah, now that it is an independent message we can do it.

Reply:
The only disadvantage of doing so is that the Seccomp gadget package will have to import the `gadgettracermanager/api` package. WDYT?

Reply:
That's not a big problem IMO.